WordPress/src/main/res/values-night/colors.xml

```diff
@@ -38,4 +38,12 @@
 
     <color name="post_history_underlying_background">@color/gray_100</color>
 
+    <!-- Reader Following empty view -->
```

Comment:
@theck13 @khaykov Quick question: What is the best approach to creating a vector drawable in light&dark modes? I noticed we mostly use two copies of the drawable and the only difference between them is the color they reference (this color is from the SEMENATIC or STUDIO sections of the colors.xml. 
The approach in this PR is to introduce new entries in the colors.xml and night/colors.xml and reference SEMANTIC or STUDIO from them. How do you feel about it?

Reply:
@malinajirka - Tyler and I discussed the approach on slack before implementation.

Reply:
Ohh perfect, thanks! ;)